README.md

```diff
@@ -9,7 +9,8 @@ $ hi \
     --module-name 'Example.Module' \
     --repository 'https://github.com/tfausak/haskeleton.git' \
     --author 'Jane Doe' \
-    --email 'jane.doe@example.com'
+    --email 'jane.doe@example.com' \
+    example-project
```

Comment:
This is only necessary because hi changed the `module-name` option to `moduleName`. You fixed that already in #26.